core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.ts

```diff
@@ -164,6 +186,30 @@ export class OperatorPropertyEditFrameComponent implements OnInit, OnChanges, On
    * @param event
    */
   onFormChanges(event: Record<string, unknown>): void {
+    const filename: string = <string>event["fileName"];
```

Comment:
This assumes "fileName" to be the only key for file names in an operator property. It might be better if you comment here to remind others.